core_backend/app/question_answer/routers.py

```diff
@@ -140,6 +142,24 @@ async def get_llm_answer(
 
         response.debug_info["extracted_info"] = rag_response.extracted_info
 
+        tts_save_path = f"response_{response.query_id}.mp3"
+
+        if question.query_metadata.get("use_tts", False):
```

Comment:
We should pass `use_tts` in the schema and not inside the metadata

Reply:
so in the `QueryBase` schema just add another optional flag right for `use_tts`?

Reply:
Yes